pylint/checkers/base/comparison_checker.py

```diff
@@ -343,15 +343,12 @@ def _check_type_x_is_y(
         ):
             return
 
-        if operator in {"is", "is not"} and _is_one_arg_pos_call(right):
```

Comment:
I think this is the only change that should be in this PR.

Reply:
https://github.com/pylint-dev/pylint/pull/10170/commits/986d1b40881d51769fa8a2c82d92e85ad361f563 is a simplification that follows from it (we're not using the operator parameter anymore)